.github/workflows/preview_comment.yaml

```
@@ -0,0 +1,70 @@
+name: "Post preview links for changed files"
+
+on:
+    pull_request:
+        paths:
+           - 'docs/**.md'
+    workflow_call:
+        inputs:
+            project:
+              description: 'The project to build (dev doc, user doc)'
+              default: ''
+              required: false
+              type: string
+
+jobs:
+    post-preview-links:
+        name: Post preview links for changed files
+        runs-on: ubuntu-latest
+        permissions:
+            # Needed to manage the comment
+            pull-requests: write
+
+        steps:
+            - uses: actions/checkout@v4
+              with:
+                  fetch-depth: 0
+
+            -   name: Create comment for changed files
+                run: |
+                    file_limit=100
+                    build_url="https://ez-systems-developer-documentation--${{ github.event.pull_request.number }}.com.readthedocs.build/${{inputs.project}}en/${{ github.event.pull_request.number }}/"
+
+                    changed_files=$(git diff --name-only HEAD "origin/$GITHUB_BASE_REF" | grep -E ".md$" || [[ $? == 1 ]])
+                    number_of__changed_files=$(echo "$changed_files" | wc -l)
+
+                    if [[ $changed_files -eq "" ]] ; then
+                      comment="Preview of modified files:: no markdown files changed, no preview needed."
```

If I modify a code sample, a preview is needed. I would reword to stick to the scope.

```suggestion
                      comment="Preview of modified files:: no Markdown change to preview."
```